### PR TITLE
Made UI Modification In Login Register and Footer

### PR DESCRIPTION
**Pull request by Tejas2909**

### CODE REVIEW

File: client/src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -11,11 +11,22 @@ const Footer = () => {
...
-            <a href="">Privacy Policy</a>
+            <a className="footerLinks" href="">
+              About
+            </a>
+            <a className="footerLinks" href="">
+              Contact
+            </a>
+            <a className="footerLinks" href="">
+              Privacy Policy
+            </a>
```
Comment (ShubhankarKG) [inline client/src/components/Footer/Footer.jsx:29]: Please update all these `<a>` with `<Link>` tags

File: client/src/components/LoginRegister/Login.jsx [hunk 2]
```diff
@@ -84,10 +95,23 @@ const Login = (props) => {
                     aria-describedby="my-helper-text"
                   />
                   <small style={{ color: "red" }}>{warningEmail}</small>
-                </FormControl>
-              </div>
-              <div className="" style={{ marginTop: "0.3rem" }}>
-                <Button
+                </FormControl> */}
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Login.jsx:98]: Please remove the comment if not needed

File: client/src/components/LoginRegister/Login.jsx [hunk 3]
```diff
@@ -84,10 +95,23 @@ const Login = (props) => {
...
+                </div>
+                <div className="" style={{ marginTop: "0.3rem" }}>
+                  <Button
+                    style={{
+                      width: "20%",
+                      // alignSelf: "left",
+                      // marginTop: "1rem",
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Login.jsx:105]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Login.jsx [hunk 4]
```diff
@@ -103,10 +127,11 @@ const Login = (props) => {
                   disabled={loading}
                 >
                   Send OTP
-                </Button>
+                </Button> */}
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Login.jsx:130]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 5]
```diff
@@ -2,9 +2,9 @@ import React, { useState, useEffect } from "react";
 import "./css/FormStyle.css";
 import {
   FormControl,
-  Input,
+  // Input,
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:5]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 6]
```diff
@@ -2,9 +2,9 @@ import React, { useState, useEffect } from "react";
...
   InputLabel,
-  Button,
+  // Button,
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:7]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 7]
```diff
@@ -176,8 +236,23 @@ const Registration = () => {
                   {props.touched.field ? (
                     <small style={{ color: "red" }}>{props.errors.field}</small>
                   ) : null}
-                </FormControl>
-                <FormControl
+                </FormControl> */}
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:239]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 8]
```diff
@@ -195,15 +270,24 @@ const Registration = () => {
...
-              </div>
-              <div
-                style={{
-                  display: "flex",
-                  justifyContent: "space-around"
-                }}
-              >
-                <FormControl
+                </FormControl> */}
+                {/* </div> */}
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:274]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 9]
```diff
@@ -227,12 +311,11 @@ const Registration = () => {
                       {props.errors.reference}
                     </small>
                   ) : null}
-                </FormControl>
-              </div>
+                </FormControl> */}
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:314]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 10]
```diff
@@ -262,8 +345,21 @@ const Registration = () => {
...
                 <Button
+                  // disabled={loading}
+                  type="submit"
+                  // onClick={onBtnClick}
+                  variant="contained"
+                  text="Register"
+                  style={{
+                    width: "20%",
+                    // alignSelf: "left",
+                    // marginTop: "1rem",
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:357]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 11]
```diff
@@ -262,8 +345,21 @@ const Registration = () => {
...
+                    marginLeft: "1.5rem"
+                  }}
+                />
+                {/* <div className=""> */}
+                {/* <Button
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:362]: Please remove the comment if not needed.

File: client/src/components/LoginRegister/Registration.jsx [hunk 12]
```diff
@@ -279,11 +375,12 @@ const Registration = () => {
...
                 >
                   Register
-                </Button>
-              </div>
-              {/* </div> */}
-            </form>
-            <small style={{ color: "red" }}>{alert}</small>
+                </Button> */}
+                {/* </div> */}
+                {/* </div> */}
```
Comment (ShubhankarKG) [inline client/src/components/LoginRegister/Registration.jsx:380]: Please remove the comment if not needed.

**Review by ShubhankarKG — CHANGES_REQUESTED** [review]

**Review by BhavyaSheth22 — COMMENTED** [review]
Screenshot from 2021-04-02 15-39-25

The home page animations are not appearing after this PR. Can you please resolve them?

**Comment by Tejas2909** [comment]
Made all the Changes

**Comment by BhavyaSheth22** [comment]
One final change, the error msg and validation for login form doesn't show up. Please rectify that

**Comment by Tejas2909** [comment]
Done!

**Review by BhavyaSheth22 — APPROVED** [review]

**Pull request merged**